client/src/App.js

```diff
@@ -31,6 +43,46 @@ function App() {
         </Col>
       </Row>
     </Container>
+  ); */
+
+  return (
+    <Router>
+      <Container>
+        <Row>
+          <NavbarComponent cart={1}/>   
+          {/*cart prop receives the length of the items list in shopping cart */}
+        </Row>
+        <Row className='vh-100 justify-content-center align-items-center'>
+          <Switch>
+            <Route exact path="/">
+              <ProducutListPage/>
```

Comment:
Typo on the page name 
```suggestion
              <ProductListPage/>
```